src/common/options.rs

```diff
@@ -53,6 +53,27 @@ pub struct Options {
         help = "Set stage1 log file name"
     )]
     log_file: Option<PathBuf>,
+    #[clap(
+        long,
+        help = "Logs to RAM and then dumps logs to balenaOS disk after flashing"
+    )]
+    fallback_log: bool,
+    #[clap(
+        long,
+        value_name = "FALLBACK_LOG_FILENAME",
+        value_parser,
+        default_value = "fallback.log",
+        help = "Set the name of the fallback log"
+    )]
+    fallback_log_filename: String,
+    #[clap(
+        long,
+        value_name = "FALLBACK_LOG_DIR",
+        value_parser,
+        default_value = "fallback_log",
+        help = "Set the directory name where fallback logs will be persisted on data partition"
```

Comment:
Same comment as in `README.md`: When it says "data partition", why would this be in a directory other than `/mnt/data/balenahup`? Are you referring to situations where the source OS is not balenaOS?